### ISSUE TITLE
Block screenshots for screen security & privacy

### ISSUE THREAD

**Issue by seifraja** [kontalk/androidclient#832]
Proposal for an additional option to be implemented inside kontalk settings to Able/Disable taking screenshots inside the application.

**Comment by daniele-athome** [comment]
Some notes on this: https://commonsware.com/blog/2016/06/06/psa-flag-secure-window-leaks.html

**Comment by OffifialMITX** [comment]
Maybe adding a builtin keyboard would solve that problem, at least partly?
This also would make it possible to customize it to Kontalk's design.

**Comment by TheLastProject** [comment]
As a Dvorak user, please no extra keyboard ^^"

**Comment by OffifialMITX** [comment]
Well, I didn't say that
1. It should be the only input method
2. There shouldn't be different layouts
But it would be nice if the keyboards would support screenshot blocking either.

**Comment by daniele-athome** [comment]
I believe screenshot protection is valid for all the screen, isn't it? Or maybe I'm misinformed?

Anyway, a dedicated input method isn't really a solution.

**Comment by OffifialMITX** [comment]
Yes and no, if you try to make a screenshot it usually don't. If you toggle screencast or record the screen the place where the app is, is just colored black but keyboard and other UI elements are still visible.

Am 8. Dezember 2016 17:38:27 MEZ, schrieb Daniele Ricci <notifications@github.com>:
